controllers/part/part_ctlr.go

```diff
@@ -77,8 +79,45 @@ func All(w http.ResponseWriter, r *http.Request, params martini.Params, enc enco
 			count = ct
 		}
 	}
+	//Convert given dates to ISO8601 format.
+	if qs.Get("modified-from") != "" {
+		from = qs.Get("modified-from")
+		_, err := time.Parse(time.RFC3339, from)
+		if err != nil {
+			_, err := time.Parse(time.RFC3339, from+"-06:00")
```

Comment:
Originally I had it as UTC, but I changed it to "-06:00" because I noticed that almost every timestamp I saw had that as the timezone. Changing it back to UTC would work for me, just wanted to try and match up with the data